src/content/runtime-apis/scheduled-event.md

```diff
@@ -0,0 +1,42 @@
+# ScheduledEvent
+
+## Background
+
+The event type for Scheduled requests to a Worker. The `Object` passed through as `event` in 
+[`addEventListener()`](/runtime-apis/add-event-listener).
+
+## Context
+
+```js
+addEventListener("scheduled", event => {
+  event.waitUntil(handleScheduled(event))
+})
+```
+
+### Properties
+
+<Definitions>
+
+- `event.type` <Type>string</Type>
+    - The type of event.
+
+- `event.scheduledTime` <Type>scheduledTime</Type>
```

Comment:
what is the `scheduledTime` type here? unix timestamp/int?

Reply:
Milliseconds since January 1, 1970, UTC to make it easy to make a `Date` with `new Date(event.scheduledTime)`.

Reply:
Added!